starter/script.js

```diff
@@ -1 +1,212 @@
+alert("You tell them of your travels. \nThey are so enamored with you and excited by your stories. \nCoachellanna: We love all that is you. Please choose a gift as token of our appreciation and affection.");
+alert("Two other merbeings come up beside Coachellanna, and each hold out a clam.\nChoose: Choose: Gain *(D)ragon Scale Dress.* \nOR: Gain *(M)edusa necklace*, \nOR: Gain *(U)nicorn Rainbow Lipstick*")
+
+
+var mermaidGift = prompt("Choose a clam: Pick '1', '2', or '3'.");
+mermaidGift = noAnswerLoop(mermaidGift);
+
+if((mermaidGift === '1') || (mermaidGift === 1)){
+    alert("Thank you, I'll choose this one.")
+    backPack['Dragon Scale Dress'] += 1;
+}else if((mermaidGift === '2') || (mermaidGift === 2)){
+    alert("Thank you, I'll choose this one.")
+    backPack['Medusa necklace'] += 1;
+}else if((mermaidGift === '3') || (mermaidGift === 3)){
+    alert("Thank you, I'll choose this one.")
+    backPack['Unicorn Rainbow Lipstick'] += 1;
+}else{
+    alert("By not picking an option the mermaids decide to suprise you with *Dragon Scale dress*");
+    backPack['Dragon Scale Dress'] += 1;
+};
+
+alert("After sunbathing for a time, you realize you need to go. \nAs you gather your things, the mermaids pull you back into the water.");
+alert("This isn't good. What will you do?");
+
+var trappedFish = prompt("Choose: 'L' for 'Lie, and say you will come back after your mission \nOR 'S' for 'Stay, have weird mermaid orgy and leave while mermaids are still cuddling and entangled in each other.'");
+trappedFish = noAnswerLoop(trappedFish);
+
+if((trappedFish === 'l') || (trappedFish === 'L')){
+    alert("You lie to the merpeople. But they believe you and give you locks of their hair so you won't forget to return.")
+    alert("You gain *Mermaid Weave* ");
+    backPack['Mermaid Weave'] += 1;
+}else {
+    alert("That fun. You didn't gain any treasure...well not the treasure Princess PEAR would want anyway. Ha!")
+
+                  //All at Phoenix
+alert("You have been walking along the Oasis' sandy beach for a while when you spot a sickly looking enormous bird.");
+alert("You walk up to the bird and it smiles. \nBird:'I am a Phoenix. My name is Hashtag Lolly, but you can call me Hash for short.'");
+alert("I am so happy you are here to keep me company in my final hours. In fact, if you guess the number tattooed on my neck, \nI will tell you the dsecret to glamour and sassyness which any admirer would love.");
+
+var guessNum = prompt("This will be easy, so you agree to Hash's game.", "Pick a number between 1 and 20.");
+guessNum = noAnswerLoop(guessNum);
+
+var hashNum = Math.floor((Math.random() * 20) +1);
+
+if(guessNum === hashNum){
+  alert("Thank you for keeping an old Phoenix company. \nThe secret is to take my ashes for the perfect smokey eye eyeshadow.");
+  alert("Hash dies leaving behind his dust. You gain *SmokeyEye by PHOENIX*.");
+  backPack['SmokeyEye by PHOENIX'] = 1;
+}
```

Comment:
One way to make this more interesting is to add some code that tells the user whether his or her guess is higher or lower than the randomly-generated number.